src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -65,8 +65,8 @@ export class ArtworkSidebarBidAction extends React.Component<
     selectedMaxBidCents: null,
   }
 
-  setMaxBid = (newVal: number) => {
-    this.setState({ selectedMaxBidCents: newVal })
+  setMaxBid = (newVal: string) => {
```

Comment:
This was never a `number` — puzzling that we are throwing away the original event type signature TBH; allows these mistakes to leak in. But apparently string/number didn't make a difference. This corrects the typing.